package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dekart",
-  "version": "0.16.0",
+  "version": "0.16.1-rc.0",
```

Comment:
Could please not bump the version, we do it in the release branch.

Reply:
Thanks @delfrrr for pointing this out. But I didn't bumped up the version, it was automatically did. Any plugins you guys have installed to take care of this ?